app/views/events/_upcoming_events.html.erb

```diff
@@ -0,0 +1,31 @@
+<section>
+  <div class="container marketing">
+    <div class="row">
+        <div class="col-lg-4">
+          <img class="img-circle"
+          src="" alt="Generic placeholder image" width="140" height="140">
+          <h2><h2>Heading</h2>
+          <p></p>
+          <p><a class="btn btn-default" href="#" role="button">Go to event »</a></p>
```

Comment:
I don't think these buttons need to be in paragraphs, but the wireframe suggests the text 'Register now'